projects/json-sanitizer/project.yaml

```diff
@@ -0,0 +1,8 @@
+homepage: "https://github.com/OWASP/json-sanitizer"
+language: jvm
+primary_contact: "mikesamuel@gmail.com"
+auto_ccs:
+  - "meumertzheim@code-intelligence.com"
+fuzzing_engines:
```

Comment:
If you only intend to fuzz with ASAN then you also need to explicitly say:
```
sanitizers:
  - address
```

Otherwise you will get a UBSAN build as well.

Reply:
Is there a way to disable sanitizers alltogether for pure JVM projects? That would be ideal, but your suggestion would also be fine.

Reply:
I think it would require too much special casing (not only in OSS-Fuzz but also in ClusterFuzz).
I think it is better to document to Jazzer fuzzers to specify address.

Reply:
Makes sense, I made the change.